### PR TITLE
Support default value when reading inexistent module attribute

### PR DESCRIPTION
**Pull request by mguimas**
This PR permits to provide a default value as the third argument to `Module.get_attribute`.

### CODE REVIEW

File: lib/elixir/lib/kernel.ex [hunk 1]
```diff
@@ -2973,15 +2973,15 @@ defmodule Kernel do
...
+          case Module.__get_attribute__(__MODULE__, unquote(name), unquote(line)) do
             {_, doc} -> doc
             other -> other
           end
         end
 
       false ->
         quote do
-          Module.get_attribute(__MODULE__, unquote(name), unquote(line))
+          Module.__get_attribute__(__MODULE__, unquote(name), unquote(line))
```
Comment (josevalim) [inline lib/elixir/lib/kernel.ex]: I like this a lot. :heart: For consistency, can you please rename the private `put_attribute` to `__put_attribute__` too?

Comment (mguimas) [reply lib/elixir/lib/kernel.ex]: I renamed it in next commit. Thanks !

File: lib/elixir/test/elixir/module_test.exs [hunk 2]
```diff
@@ -436,4 +436,30 @@ defmodule ModuleTest do
...
+      in_module do
+        Module.register_attribute(__MODULE__, :value, accumulate: true)
+        Module.put_attribute(__MODULE__, :value, 1)
+        assert Module.get_attribute(__MODULE__, :value) == [1]
+        Module.put_attribute(__MODULE__, :value, 2)
+        assert Module.get_attribute(__MODULE__, :value) == [2, 1]
+      end
+    end
+
+    test "returns the value of the attribute if this exists" do
```
Comment (fertapric) [inline lib/elixir/test/elixir/module_test.exs]:
```suggestion
    test "returns the value of the attribute if it exists" do
```

File: lib/elixir/test/elixir/module_test.exs [hunk 3]
```diff
@@ -436,4 +436,30 @@ defmodule ModuleTest do
       Module.get_attribute(Enum, :moduledoc)
     end
   end
+
+  describe "get_attribute/3" do
+    test "returns a list when the attribute is marked as `accummulate`" do
```
Comment (fertapric) [inline lib/elixir/test/elixir/module_test.exs]:
```suggestion
    test "returns a list when the attribute is marked as `accummulate: true`" do
```

**Pull request merged**